guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -110,15 +109,15 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
     }
 
     saveToSnippetServer() {
-        var adt = this.props.globalState.guiTexture;
+        let adt = this.props.globalState.guiTexture;

```

Comment:
Most of the `let` you added could be `const` instead :)

If in doubt, just always use `const` and change to `let` if the transpiler does complain.